sync/course-db.js

```diff
@@ -680,26 +687,35 @@ module.exports.loadInfoFile = async function ({
  * @returns {Promise<InfoFile<Course>>}
  */
 module.exports.loadCourseInfo = async function (coursePath) {
-  /** @type {import('./infofile').InfoFile<Course>} */
-  const loadedData = await module.exports.loadInfoFile({
+  /** @type {import('./infofile').InfoFile<Course> | null} */
+  const maybeNullLoadedData = await module.exports.loadInfoFile({
     coursePath,
     filePath: 'infoCourse.json',
     schema: schemas.infoCourse,
   });
-  if (infofile.hasErrors(loadedData)) {
+
+  if (maybeNullLoadedData && infofile.hasErrors(maybeNullLoadedData)) {
     // We'll only have an error if we couldn't parse JSON data; abort
-    return loadedData;
+    return maybeNullLoadedData;
+  }
+
+  if (!maybeNullLoadedData || !maybeNullLoadedData.data) {
+    throw new Error('Could not load infoCourse.json');
   }
```

Comment:
I'm *pretty* sure that we just want to blow up in this case? We didn't previously handle the case where `loadedData` was null.

Reply:
I can't think of any other useful action to take?